<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GxsCommentContainer</class>
 <widget class="QWidget" name="GxsCommentContainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>557</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <item>
    <widget class="QFrame" name="titleBarFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="titleBarFrameHLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="titleBarPixmap">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="pixmap">
         <pixmap resource="../Posted/Posted_images.qrc">:/images/posted_24.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="StyledLabel" name="titleBarLabel">
        <property name="text">
         <string>Comment Container</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="titleHSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>573</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="RSTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="tabsClosable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StyledLabel</class>
   <extends>QLabel</extends>
   <header>gui/common/StyledLabel.h</header>
  </customwidget>
  <customwidget>
   <class>RSTabWidget</class>
   <extends>QTabWidget</extends>
   <header>gui/common/RSTabWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Posted/Posted_images.qrc"/>
 </resources>
 <connections/>
</ui>
